client/support/support-user/active-dialog.jsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * External Dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal Dependencies
+ */
+import Dialog from 'components/dialog';
+import FormButton from 'components/forms/form-button';
+import FormLabel from 'components/forms/form-label';
+import EnableLocaleCheckbox from './enable-locale-checkbox';
+import config from 'config';
+import { getCurrentUser } from 'state/current-user/selectors';
+import i18n from 'lib/mixins/i18n'
+
+class SupportUserActiveDialog extends Component {
+
+	render() {
+		const { isVisible, onCloseDialog, onRestoreUser } = this.props;
+
+		const { defaultLocaleSlug, userLocaleSlug } = this.props;
+
+		const buttons = [
+			<FormButton
+				key="close"
+				isPrimary={ false }
+				onClick={ onCloseDialog }>
+					Close
```

Comment:
This is a good point. There's an interesting side-effect of translating these strings though, it's a bit of a confusing problem. When the interface is displayed in the supported user's locale, the support user interface will also change to that locale. Assuming they're a seasoned "support user user", they'll already know where everything is and still be able to use the interface. But a first time "support user user" might not be able to find the 'Restore User' button because it could be in Chinese :)